kubetest/main.go

```diff
@@ -459,7 +487,7 @@ func prepare() error {
 		return err
 	}
 
-	if err := os.MkdirAll("./_artifacts", 0777); err != nil { // Create artifacts
+	if err := os.MkdirAll(artifacts, 0777); err != nil { // Create artifacts
```

Comment:
The main thing I'm trying to address here the same as in https://github.com/kubernetes/test-infra/pull/2024, namely that `--extract=E` will cause kubetest to CD to a different dir. So when I `chmod -R o+r _artifacts` at the end of the run I need this path to include the PWD at the time it was created.